be/src/exec/json_parser.cpp

```diff
@@ -40,6 +47,75 @@ Status JsonDocumentStreamParser::parse(char* data, size_t len, size_t allocated)
     return Status::OK();
 }
 
+bool JsonDocumentStreamParser::_check_and_new_doc_stream_iterator() {
+    size_t cur_left_len = _first_object_parsed ? _len - _last_begin_offset : _len;
+    if (_batch_size >= cur_left_len) {
```

Comment:
Will _batch_size shrink sometime after having parsed a large object?

Reply:
_batch_size can be Increased only